tests/providers/cncf/kubernetes/operators/test_pod.py

```diff
@@ -28,6 +28,7 @@
 from urllib3 import HTTPResponse
 
 from airflow.exceptions import AirflowException, AirflowSkipException, TaskDeferred
+from airflow.kubernetes import pod_generator
```

Comment:
```suggestion
from airflow.providers.cncf.kubernetes import pod_generator
```

To fix the import error
after that it requires also to rearrange order of imports